Lib/test/test_perf_profiler.py

```diff
@@ -0,0 +1,153 @@
+import unittest
+import subprocess
+import re
+import sys
+import sysconfig
+import os
+from test import support
+from test.support.script_helper import make_script
+from test.support.os_helper import temp_dir
+
+
+if not support.has_subprocess_support:
+    raise unittest.SkipTest("test module requires subprocess")
+
+
+def is_unwinding_reliable():
+    cflags = sysconfig.get_config_var("PY_CORE_CFLAGS")
+    if not cflags:
+        return False
+    return "no-omit-frame-pointer" in cflags
+
+
+if not is_unwinding_realiable():
```

Comment:
```suggestion
if not is_unwinding_reliable():
```
Sorry, I should have mentioned this one too in comment above.